java/client/src/main/java/glide/api/commands/TransactionsCommands.java

```diff
@@ -0,0 +1,42 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.commands;
+
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Supports commands for the "Transactions Commands" group for a standalone client.
+ *
+ * @see <a href="https://redis.io/commands/?group=transactions">Transactions Commands</a>
+ */
+public interface TransactionsCommands {
```

Comment:
Rename to follow the style, because it goes to the base client
```suggestion
public interface TransactionsBaseCommands {
```